src/App.tsx

```diff
@@ -61,8 +77,24 @@ function Welcome(_props: RouteComponentProps): JSX.Element {
   return (
     <div css={c.welcome}>
       <Hello compiler="TypeScript" framework="React" />
-      <button onClick={() => navigate('/')}>Home</button>
       <Search />
+      <Button variant="contained" color="primary" onClick={() => navigate('/')}>
+        Home
+      </Button>
```

Comment:
Nice! I made almost the exact same change in my CL. I'm glad you figured this out.